packages/hash/frontend/src/components/layout/PageHeader/PageHeader.tsx

```diff
@@ -1,17 +1,15 @@
 import React from "react";
-// import { useModal } from "react-modal-hook";
-import { Box, Button, useTheme, useMediaQuery } from "@mui/material";
-import { useRouter } from "next/router";
+import { Box, useTheme, useMediaQuery } from "@mui/material";
 
 import { useLogout } from "../../hooks/useLogout";
 import { useUser } from "../../hooks/useUser";
-// import { LoginModal } from "../../Modals/AuthModal/LoginModal";
 import { AccountDropdown } from "./AccountDropdown";
 import { SearchBar } from "./SearchBar";
 import { HashNavIcon } from "../../icons";
 import { Link } from "../../Link";
 import { ActionsDropdown } from "./ActionsDropdown";
 import { NotificationsDropdown } from "./NotificationsDropdown";
+import { LinkButton } from "../../LinkButton";
 
 const Nav: React.FC = ({ children }) => (
```

Comment:
According to the designs at no point should these icons have a blue outline:
<img width="168" alt="image" src="https://user-images.githubusercontent.com/42802102/158066438-6ccb27c5-099b-4f2b-b6ce-93c4bb7f8a7f.png">
<img width="193" alt="image" src="https://user-images.githubusercontent.com/42802102/158066440-3e56ee0f-0567-4d14-b305-a2e770dd849b.png">

Reply:
This is matching the focus styles from the design: https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-Editor?node-id=179%3A3548.

This is the piece of code causing the outline to appear:

```js
"&:focus": {
  outlineColor: theme.palette.blue[70],
  outlineOffset: "2px",
},
```

Let me know if this isn't expected behavior, or if you would do this in a different way.

Reply:
I'm aware that the default icon buttons should have a blue outline.

Perhaps I've misunderstood the designs, but I would've assumed that the hover/focus states made for the `Navbar` would supercede the ones made for all icon buttons?
<img width="379" alt="image" src="https://user-images.githubusercontent.com/42802102/158149908-1dfeb88d-0044-48b1-8a0b-26007f7a41e4.png">

Reply:
Right, this seems to be a later addition as well. Added this here: https://github.com/hashintel/hash/pull/397/commits/edcb65887888215a59df151f9ef97ce0566a56b6.

If we scroll down [from here](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-Editor?node-id=179%3A3591) on the Figma, it looks like the actions button doesn't have the gray outline when it's open, but the accounts dropdown does. Hence, I've implemented it here similarly.

Reply:
@benwerner01, you might already know, but they actually have a blue outline when focused [figma](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-Editor?node-id=179%3A4588). @akash-joshi I don't think you need to address this since I'll handle this in this [branch](https://github.com/hashintel/hash/tree/vu/sidebar-updates-2)

Reply:
I'm not sure this is resolved as I think the hover/focus effects should be the same for the icon buttons and the notification button.

https://user-images.githubusercontent.com/42802102/159002510-a1b7b676-365c-4c7a-83c5-40f83297164a.mov

I appreciate this isn't explicitly stated in the designs, so maybe worth double checking with Maggie.

You could create a task for this and address it in follow-up, but I'm not sure that makes sense given this PR isn't high priority and isn't blocking other PRs (as far as I'm aware) - so getting it right now is going to save us time in the long run.